### PR TITLE
13.0 mig helpdesk type

### PR DESCRIPTION
**Pull request by AntoniRomera**

### CODE REVIEW

**Review by MRomeera — APPROVED** [review]
Functional test ok.

**Review by FernandoRomera — APPROVED** [review]
Functional test ok.

**Comment by FernandoRomera** [comment]
@pedrobaeza @DarioLodeiros Can you merge this PR?.

File: helpdesk_type/models/helpdesk_ticket.py [hunk 1]
```diff
@@ -11,5 +11,6 @@ class HelpdeskTicket(models.Model):
 
     @api.onchange("type_id")
     def _onchange_type_id(self):
-        self.team_id = False
-        self.user_id = False
+        for record in self:
```
Comment (pedrobaeza) [inline helpdesk_type/models/helpdesk_ticket.py]: Why are you changing this? `onchange` are per definition executed per record

Comment (AntoniRomera) [reply helpdesk_type/models/helpdesk_ticket.py]: Done

**Comment by pedrobaeza** [comment]
Please squash the last 2 commits into one, as the second is not significant enough to be kept (and more taking into account that is undoing a previous change).

**Comment by AntoniRomera** [comment]
Done

**Pull request merged**